ubyssey/templates/objects/archive.html

```diff
@@ -0,0 +1,198 @@
+<div class="o-archive">
+
+  <div class="o-archive__search">
+    <form method="get">
+      <label class="o-archive__search__label" for="c-articles-list__searchbar"><i class="fa fa-search"></i></label>
+      <input class="o-archive__search__input" name="q" id="c-articles-list__searchbar" type="text" {% if q %}value="{{ q }}"{% endif %} placeholder="Search" />
+    </form>
+  </div>
+
+  <div class="o-archive__container">
+
+    {% if sections or years %}
+    <div class="o-archive__sidebar show-desktop">
+
+      <div class="o-archive__sidebar__filters">
+        <h3>Year</h3>
+        <ul>
+          <li class="o-archive__sidebar__filter{% if not year %} o-archive__sidebar__filter--is-active{% endif %}">
+            <a href="{% if q and section_id %}?q={{ q }}&section_id={{ section_id }}{% elif q %}?q={{ q }}{% elif section_id %}?section_id={{ section_id }}{% endif %}">All years</a>
```

Comment:
[join](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#join) also exists and should work
```html
<a href="{% if q or section_id %}?{% endif %}{{ ['q='+q, 'section_id'+section_id] | join:'&' }}">